src/components/Controls/Controls.tsx

```diff
@@ -52,6 +52,7 @@ export default function Controls(props: { disabled?: boolean }) {
       <ToggleVideoButton disabled={isReconnecting} />
       {roomState !== ROOM_STATE.DISCONNECTED && (
         <>
+          {/* <ToggleScreenShareButton disabled={isReconnecting} /> */}
```

Comment:
Is this a part of the current commit?
Do we have another task for this one?

Reply:
I will open a new story  on this, for now, its not render